go/src/meguca/websockets/post_updates.go

```diff
@@ -216,10 +216,26 @@ func (c *Client) _closePost() (err error) {
 	if err != nil {
 		return
 	}
+
+	CheckRouletteBan(com, c.post.board, c.post.id)
+
 	c.post = openPost{}
 	return
 }
 
+func CheckRouletteBan(commands []common.Command, board string, id uint64) {
+	// Meme ban if the poster lost at #roulette
+	for _, command := range commands {
+		if command.Type == common.Roulette {
+			if command.Roulette[0] == 1 {
+				// don't bother error checking
+				db.Ban(board, "lost at #roulette", "system", time.Now().Add(time.Second*30), id)
```

Comment:
Sorry, If I wasn't clear. I mean that you should check for errors almost always. Especially I/O errors.

Reply:
What should I do if there is an error here?
I don't think it makes sense to cancel the post and return an error to the client just because the memeban failed.
Maybe just log it?

Reply:
The memeban can only fail on database write failure and database write failure is a pretty serious error. You should definitely return it up the call stack.